Dispatcher/Supporting Files/Utils.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Utils.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 04/05/2023.
+//
+
+import Foundation
+
+struct Utils {
```

Comment:
we don't really have to create a struct in order to have util functions. We can just write "func formatDate" directly in the swift file and spare the struct